setup.cfg

```diff
@@ -101,7 +101,7 @@ install_requires =
     flask-session>=0.4.0
     flask-wtf>=0.15
     google-re2>=1.0
-    graphviz>=0.12
+    #graphviz>=0.12
```

Comment:
Leftover? Or is it intended to test things out?

Reply:
yeah. absolutely